<launch>

    <arg name="markerSize"      default="0.05"/>    <!-- in m -->
    <!--<arg name="side"             default="left"/>-->
    <arg name="ref_frame"       default="base_link"/>  <!-- leave empty and the pose will be published wrt param parent_name -->
    <arg name="marker_frame"    default="aruco_marker_frame"/>


    <node pkg="aruco_ros" type="marker_publisher" name="aruco_marker_publisher" output="screen">
        <remap from="/camera_info" to="/camera/color/camera_info" />
        <remap from="/image" to="/camera/color/image_raw" />
        <param name="image_is_rectified" value="True"/>
        <param name="marker_size"        value="$(arg markerSize)"/>
        <param name="reference_frame"    value="$(arg ref_frame)"/>   <!-- frame in which the marker pose will be refered -->
        <param name="camera_frame"       value="camera_color_optical_frame"/>
        <param name="marker_frame"       value="$(arg marker_frame)" />

    </node>

</launch>

